osu.Desktop/Overlays/VersionManager.cs

```diff
@@ -154,31 +144,45 @@ protected override void Dispose(bool isDisposing)
                     //no updates available. bail and retry later.
                     return;
 
-                if (notification == null)
+                if (notificationContainer == null)
                 {
-                    notification = new UpdateProgressNotification { State = ProgressNotificationState.Active };

```

Comment:
honestly this feels worse than before. now you're having to pollute the local class with design-level knowledge of the drawable notification. and this isn't just the case here. every time a notification wants to be coloured this is going to be the case.